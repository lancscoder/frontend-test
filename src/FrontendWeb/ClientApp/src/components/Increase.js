import React from 'react';
import Glyph from './Glyph';

const Increase = () => (
  <Glyph name="glyphicon glyphicon-arrow-up icon-increase" />
);

export default Increase;