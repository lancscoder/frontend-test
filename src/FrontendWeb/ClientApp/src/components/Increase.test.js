import React from 'react';
import renderer from 'react-test-renderer';
import Increase from './Increase';

describe('Increase ::', () => {
    it('should render with correct class', () => {
        const component = renderer.create(<Increase />);;

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

        expect(tree.props.className).toBe('glyphicon glyphicon-arrow-up icon-increase');
    });
});