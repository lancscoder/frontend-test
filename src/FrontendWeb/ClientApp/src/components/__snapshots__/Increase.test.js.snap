// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Increase :: should render with correct class 1`] = `
<span
  aria-hidden="true"
  className="glyphicon glyphicon-arrow-up icon-increase"
  style={
    Object {
      "paddingLeft": "5px",
      "paddingRight": "5px",
    }
  }
/>
`;
